### ISSUE TITLE
How to prepare the input Hi-C data?

### ISSUE THREAD

**Issue by shenscore** [ChristopherBarrington/domainClassifyR#1]
I found that there is many negetive values in the example contact data. I wonder how to prepare the input Hi-C data from contact matrix.

>   chrom1   start1     end1 chrom2   start2     end2
1  chr19 30683824 30683825  chr19 45631573 45631574
2  chr19 30617472 30617473  chr19 45606509 45606510
3  chr19 30669904 30669905  chr19 45606508 45606509
4  chr19 30694599 30694600  chr19 45644855 45644856
5  chr19 30438239 30438240  chr19 45414363 45414364
6  chr19 30470215 30470216  chr19 45389373 45389374
  hic.example_project.example_dataset.s... intervalID
1                                    -33.8        221
2                                    -28.8        221
3                                    -30.8        221
4                                    -32.8        221
5                                    -37.3        221
6                                    -35.3        221

**Comment by ChristopherBarrington** [comment]
I've not looked at this data for a long time. But these contact data are scaled KS statistics (I think) from the [shaman](https://tanaylab.github.io/shaman/) package from the Tanay lab (I'm not sure if that has now been published). The above example matrix is computed from an observed input contact matrix.

**Comment by shenscore** [comment]
> I've not looked at this data for a long time. But these contact data are scaled KS statistics (I think) from the [shaman](https://tanaylab.github.io/shaman/) package from the Tanay lab (I'm not sure if that has now been published). The above example matrix is computed from an observed input contact matrix.

Yes, you are right. The contact data are generated by the shaman package. However, this package is not available now. It seems that shaman is necessary for the domainClassify analysis.
